software/messaging/src/main/resources/catalog.bom

```diff
@@ -25,6 +25,56 @@ brooklyn.catalog:
         type: org.apache.brooklyn.entity.messaging.activemq.ActiveMQBroker
         name: ActiveMQ Broker
         description: ActiveMQ is an open source message broker which fully implements the Java Message Service 1.1 (JMS)
+    - id: org.apache.brooklyn.entity.messaging.activemq-artemis
+      version: "1.0.0-SNAPSHOT" # BROOKLYN_VERSION
+      item:
+        type: org.apache.brooklyn.entity.software.base.VanillaSoftwareProcess
+        name: ActiveMQ Artemis
+        itemType: entity
+        description: Apache ActiveMQ Artemis has a proven non blocking architecture. It delivers outstanding performance. JMS 2.0 and 1.1 support.
```

Comment:
Might be worth a note that it is a message broker, maybe

"Apache ActiveMQ Artemis is an open source message broker and has a proven non blocking architecture. It delivers outstanding performance. JMS 2.0 and 1.1 support."